code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -33,7 +33,7 @@
 	icon_state = "trashbag"
 	item_state = "trashbag"
 
-	w_class = 4
+	w_class = 3

```

Comment:
As I said, right now, if you want to wear the Janibelt like, well, a belt, you either carry your trash bag in your hand, everywhere you go, or you are forced to bring around the cumbersome and easily-stolen janitorial cart, because the bag fits there.

This lets you keep the janibelt as a belt AND carry around what is essentially a plastic bag with you in a convenient manner.